### ISSUE TITLE
Merge Blocks May (Again) Have Crashed DX6

### ISSUE THREAD

**Issue by subbob** [DraconisCluster/DX-Issues#17]
**TL;DR**
Player opened a ticket and suspects merge block operations may have caused DX6 server crash based upon timing of the event.

**Ticket Report**
Server Name: Expanse
Instance(if applicable): DX6
Steam ID: REDACTED
Grid name(if applicable): REDACTED
time, date(If applicable/grid related issues): 1/30/2024 3:04 PM PST
Purpose of the Ticket: I think i may have inadvertently crashed the server. I printed a ship that had an attached skiff. When disconnecting the skiff by turning off the merge blocks my game crashes and the server crashes. I am not aware of the reason that happened nor if the build itself if in violation of the server rules. I greatly apologize if that is the case

**Screenshots**
--If applicable, add screenshots to help explain your problem.--
**References**
[Former Ticket (closed) on Same Issue](https://github.com/DraconisCluster/DX-Issues/issues/6)
[[SDX] Icehauler Thunderbird Mk I](https://steamcommunity.com/sharedfiles/filedetails/?id=3039428628)

**Comment by subbob** [comment]
![image](https://github.com/DraconisCluster/DX-Issues/assets/5113030/914c9aea-4107-49c6-be35-231bb42cdda7)
Follow up from the user

**Comment by subbob** [comment]
![image](https://github.com/DraconisCluster/DX-Issues/assets/5113030/e09bfe93-ff78-4c45-a466-5185c9a9888d)

From a ticket submitted this morning.

**Comment by draconb** [comment]
@Terif1 merging still crashing servers

**Comment by draconb** [comment]
@SugerGlider12 was able to crash when trying to merge 2 grids (neither set to station)

**Comment by Synirrr** [comment]
Do we have logs or is it a silent crash? We need logs in order to do anything.

**Comment by SugerGlider12** [comment]
I didn’t think to grab log but know how to reproduce it

**Comment by Terif1** [comment]
basically without a stacktrace we can't do much to help fix this
